dependencies.env

```diff
@@ -1 +1 @@
-FFI_VERSION="0.19.0"
+FFI_VERSION="0.20.0"
```

Comment:
Is the new version of FFI was deployed to AWS S3?

Reply:
Not yet, currently pending.

Reply:
Version has been updated to 0.21.0.